corehq/apps/mobile_auth/management/commands/populate_mobileauthkeyrecord.py

```diff
@@ -0,0 +1,33 @@
+from dimagi.utils.dates import force_to_datetime
+
+from corehq.apps.cleanup.management.commands.populate_sql_model_from_couch_model import PopulateSQLCommand
+
+
+class Command(PopulateSQLCommand):
+    @classmethod
+    def couch_doc_type(self):
+        return 'MobileAuthKeyRecord'
+
+    @classmethod
+    def sql_class(self):
+        from corehq.apps.mobile_auth.models import SQLMobileAuthKeyRecord
+        return SQLMobileAuthKeyRecord
+
+    @classmethod
+    def commit_adding_migration(cls):
+        return "TODO"
+
+    def update_or_create_sql_object(self, doc):
+        # Use get_or_create so that if sql model exists we don't bother saving it,
+        # since these models are read-only
+        model, created = self.sql_class().objects.get_or_create(
+            uuid=doc['_id'],
```

Comment:
Since the couch ids are strings and the sql uuids are UUID objects, `doc['_id'] == obj.uuid` is false for equivalent models, but `SQLMobileAuthKeyRecord.get(uuid=doc['_id'])` does get the proper object.